apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -7,6 +7,59 @@ import (
 	apiv1alpha1 "github.com/os-observability/tempo-operator/apis/tempo/v1alpha1"
 )
 
+// BuiltInCertManagement is the configuration for the built-in facility to generate and rotate
+// TLS client and serving certificates for all Tempo services and internal clients except
+// for the tempo-gateway.
+type BuiltInCertManagement struct {
+	// CACertValidity defines the total duration of the CA certificate validity.
```

Comment:
what is the default value?

Reply:
The default value is what is provided on the `controller_manager_config` actually I think if you don't set anything it will be zero.

Reply:
> LGTM, just some minor comments.
> 
> How is handled the use case when a secret with the cert is removed by user? The jaeger-operator is caching the certs on its filesystem and reconstracts the secret.



> LGTM, just some minor comments.
> 
> How is handled the use case when a secret with the cert is removed by user? The jaeger-operator is caching the certs on its filesystem and reconstracts the secret.



> LGTM, just some minor comments.
> 
> How is handled the use case when a secret with the cert is removed by user? The jaeger-operator is caching the certs on its filesystem and reconstracts the secret.

The certificates will be regenerated using the same CA and the parameters of the expiration, host name etc..